dissect/target/tools/shell.py

```diff
@@ -1069,6 +1067,12 @@ def main():
 
     process_generic_arguments(args)
 
+    # For the shell tool we want -q to log slightly more then just CRITICAL
+    # messages.
+    dissect_logger = logging.getLogger("dissect")
+    if args.quiet:
+        dissect_logger.setLevel(level=logging.ERROR)
```

Comment:
```suggestion
    if args.quiet:
        logging.getLogger("dissect").setLevel(level=logging.ERROR)
```
I believe this could just be one line in the if block?